app/helpers/latest_news_helper.rb

```diff
@@ -0,0 +1,39 @@
+module LatestNewsHelper
```

Comment:
This new file doesn't seem to have any tests - can some be added?

Reply:
I think @KludgeKML would be best placed for this as he wrote the code for this as well - but if he's busy I can try